.gitignore

```diff
@@ -62,3 +62,6 @@ myTestCheckpoints
 *.config
 *.keytab
 .DS_Store
+
+package-lock.json
```

Comment:
In Spline, we have `package-lock.json` as part of the project. For more refer to [spline issue 237](https://github.com/AbsaOSS/spline/issues/237). There is even a nice [article](https://medium.com/coinmonks/everything-you-wanted-to-know-about-package-lock-json-b81911aa8ab8) that explains why to commit `package-lock.json`

Reply:
Depending on the action here we should handle preload.js in the #453 as well

Reply:
Fair enough